plugins/destination/postgresql/client/spec.go

```diff
@@ -8,8 +8,8 @@ import (
 
 const (
 	defaultBatchSize      = 10000
-	defaultBatchSizeBytes = 1000000
-	defaultBatchTimeout   = 10 * time.Second
+	defaultBatchSizeBytes = 100000000
```

Comment:
add doc change as well

Reply:
Done in https://github.com/cloudquery/cloudquery/pull/13324/commits/8e4f613b79aee3ff25c4d20d4870f7fe7b7c700b